R/build-mock-url.R

```diff
@@ -79,9 +79,11 @@ get_string_request_body <- function(req) {
         if (inherits(x, "form_file")) {
           # hash the file contents
           paste("File:", digest(x$path, serialize = FALSE, file = TRUE))
-        } else {
-          # assume form_data
+        } else if (inherits(x, "form_data")) {
           rawToChar(x$value)
+        } else {
+          # assume character string or raw
```

Comment:
I suspect we'll need to rawToChar if it is raw, given that this function says it is returning a string. But adding to the test in test-mock-api.R should confirm that one way or another.

Reply:
The new version confirms that rawToChar() is not needed